mountpoint-s3/tests/fstab/basic_checks.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
```

Comment:
Have we tried writing these tests with Rust? Not sure how hard would it be but at least we could re-use some existing utilities and run more exhaustive tests. `mountpoint-s3/tests/fork_test.rs` might be a good inspiration

Reply:
I considered it, but the fact we need to invoke `systemd-fstab-generator` and interact with systemd a bunch made me decide against it.